src/main/java/com/sureit/admin/attendance/model/TimeSheetEntry.java

```diff
@@ -12,9 +12,9 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 @Entity
-@Table(name = "employee_login")
+@Table(name = "timesheet_entry")
 @JsonIgnoreProperties({ "hibernateLazyInitializer", "handler" })
-public class EmployeeLogin {
+public class TimeSheetEntry {
 
 	@Id
 	@GeneratedValue(strategy = GenerationType.AUTO)

```

Comment:
No need to have loginID which is extra field in table.
If there is any constraint on having primary keys then make empid + loginDateTime as a composite primary key.